File: lab7/src/components/post/Post.js
import React from 'react'

const Post = (props) => {
  return (
    <div className="Content" onClick={props.setSelected}>
      <h3>{props.id}</h3>
      <h3>{props.title}</h3>
      <h3>{props.author}</h3>
    </div>
  )
}

export default Post

File: lab1/lab1/src/main/java/miu/edu/lab1/entity/Post.java
package miu.edu.lab1.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
//@AllArgsConstructor
public class Post {
    long id;
    String title;
    String content;
    String author;

    public Post(long id,String title,String content,String author){
        this.id=id;
        this.title=title;
        this.content=content;
        this.author=author;
    }

}
